sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -942,6 +957,47 @@ class Analyzer(
       }
   }
 
+  /**
+   * Replace [[UnresolvedRelation]] with concrete streaming logical plans.
+   */
+  object ResolveStreamingRelation extends Rule[LogicalPlan] {
```

Comment:
shall we merge it into `ResolveRelations` to avoid duplicated code?

Reply:
Done in 305c316.